e2e/main.e2e.test.ts

```diff
@@ -31,340 +43,347 @@ describe("KFC e2e test", () => {
     await waitForRunningStatusPhase(kind.Pod, { metadata: { name: namespace, namespace } });
   });
 
-  it("Apply", async () => {
-    // No Force Test - NS is already created
-    try {
-      const ns = await K8s(kind.Namespace).Get(namespace);
-      expect(ns.metadata!.name).toBe(namespace);
-    } catch (e) {
-      expect(e).toBeDefined();
-    }
-    // Force - Update NS with label
-    try {
-      await K8s(kind.Namespace).Apply(
-        {
-          metadata: {
-            name: namespace,
-            labels: {
-              "e2e-test": "true",
-            },
+  it("Scales Deployment", async () => {
+    await K8s(kind.Deployment).InNamespace(namespace).Scale(3, `${namespace}-scale`);
+    const deployment = await K8s(kind.Deployment).InNamespace(namespace).Get(`${namespace}-scale`);
+    expect(deployment.spec?.replicas).toBe(3);
```

Comment:
```suggestion
    const before = await K8s(kind.Deployment).InNamespace(namespace).Get(`${namespace}-scale`);
    expect(before.spec?.replicas).toBe(1);
    await K8s(kind.Deployment).InNamespace(namespace).Scale(3, `${namespace}-scale`);
    const after = await K8s(kind.Deployment).InNamespace(namespace).Get(`${namespace}-scale`);
    expect(after.spec?.replicas).toBe(3);
```

While we init the deployment with one replica, I think it's thorough to make sure there's one replica present before we scale it.